test/test_cli.py

```diff
@@ -217,3 +219,20 @@ def test_unknown_warning(self):
         warnings = [i for i in cm.output if "WARNING" in i]
         self.assertTrue(len(warnings) > 0)
         self.assertTrue("was detected with version UNKNOWN" in "".join(warnings))
+
+    def test_quite_mode(self):
```

Comment:
should be "quiet" not "quite" here.

Reply:
oops typo

Reply:
Still waiting on this one to be fixed so I can merge!